pages/staff.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { GetStaticProps } from "next";
+import Layout from "@/components/Layout";
+
+const Staff = () => {
+  return (
+    <Layout>
+      <div className="page">
+        <h1>Employees</h1>
+        <div className="post">{/* Content of each book post */}</div>
+      </div>
+      <style jsx>{`
+        .post {
+          background: white;
+          transition: box-shadow 0.1s ease-in;
+          /* Other styling properties */
+        }
+
+        .post:hover {
+          box-shadow: 1px 1px 3px #aaa;
+        }
+
+        .post + .post {
+          margin-top: 2rem;
+        }
+      `}</style>
```

Comment:
I'd get rid of these style tags, but for now it's probably ok

Reply:
ya, once y'all agree on a styling standard this can be reformatted but this is fine for now!